backend/src/question-service/routes/questions.ts

```diff
@@ -13,6 +13,6 @@ router.get("/categories", QuestionsController.getQuestionCategories);
 
 router.get("/:questionId", QuestionsController.getQuestion);
 
-router.put("/:questionId/update", QuestionsController.updateQuestion);
+router.patch("/:questionId/update", QuestionsController.updateQuestion);
```

Comment:
`router.patch("/:questionId", QuestionsController.updateQuestion);`

there shouldn't be `/update`